AmplifyPlugins/Auth/Tests/AuthHostedUIApp/AuthHostedUIAppUITests/Screen/SignInScreen.swift

```diff
@@ -46,9 +46,9 @@ struct SignInScreen: Screen {
     }
 
     func signIn(username: String, password: String) -> Self {
-        _ = app.webViews.textFields["Username"].waitForExistence(timeout: 60)
-        app.webViews.textFields["Username"].tap()
-        app.webViews.textFields["Username"].typeText(username)
+        _ = app.webViews.textFields["Email Email"].waitForExistence(timeout: 60)
```

Comment:
the test was failing because it could not find the textField "Username". Based the test input with the username being an email, i provisioned the backend with email as sign in. And visually, the hostedUI site has text field "Email". Upon debugging into the `app.webViews.textFields`, the key is called "Email Email"